Installation/cmake/modules/UseCGAL.cmake

```diff
@@ -26,20 +26,6 @@ if(NOT USE_CGAL_FILE_INCLUDED)
   include(${CMAKE_CURRENT_LIST_DIR}/CGAL_GeneratorSpecificSettings.cmake)
   include(${CMAKE_CURRENT_LIST_DIR}/CGAL_TweakFindBoost.cmake)
 
-  set(CGAL_INSTALLED_SCM_BRANCH_NAME ${CGAL_SCM_BRANCH_NAME})
-  set(CGAL_SCM_BRANCH_NAME "")
-
-  if( NOT "${CGAL_INSTALLED_SCM_BRANCH_NAME}" STREQUAL "" )
-    include(${CMAKE_CURRENT_LIST_DIR}/CGAL_SCM.cmake)
-    CGAL_detect_git(${CMAKE_SOURCE_DIR})
-    if ( NOT "${CGAL_SCM_BRANCH_NAME}" STREQUAL "" )
-      message ( STATUS "Code taken from Git branch: ${CGAL_SCM_BRANCH_NAME}" )
-      if ( NOT "${CGAL_SCM_BRANCH_NAME}" STREQUAL "${CGAL_INSTALLED_SCM_BRANCH_NAME}")
-        message (AUTHOR_WARNING "Branch '${CGAL_SCM_BRANCH_NAME}' does not match branch '${CGAL_INSTALLED_SCM_BRANCH_NAME}' from which CGAL has been installed. Please consider rebuilding CGAL from this branch.")
-      endif()
-    endif()
-  endif()
-
```

Comment:
Alien changes?

Reply:
Yes and no. I did it on purpose. There are several thinks in this branch (changes to Mesh_3, removal of annoying warnings in the polyhedron demo, and small changes to CMake... and maybe other things I forgot). I plan to integrate everything at once in CGAL-4.13. If necessary, we could cherry-pick the changes one by one in dedicated pull-requests. ... if necessary.